packages/jest-runtime/src/index.ts

```diff
@@ -898,64 +899,75 @@ class Runtime {
     }
 
     if (moduleName === 'module') {
-      const createRequire = (modulePath: string | URL) => {
-        const filename =
-          typeof modulePath === 'string'
-            ? modulePath.startsWith('file:///')
-              ? fileURLToPath(new URL(modulePath))
-              : modulePath
-            : fileURLToPath(modulePath);
-
-        if (!path.isAbsolute(filename)) {
+      return this._getMockedNativeModule();
```

Comment:
ignoring whitespace makes this diff readable

https://github.com/facebook/jest/pull/9711/files?w=1